REQUIREMENTS.md

```diff
@@ -15,7 +15,7 @@ The following python packages are required by the Arkouda client package.
 - `python>=3.8`
 - `numpy>=1.22.2`
 - `pandas>=1.4.0`
-- `pyzmq>=20.0.0`
+- `pyzmq==23.2.1`
```

Comment:
Do you want this version exactly, or just something >= 20 and < 24? (I'm worried that for instance pypi mirrors we use internally might not have this exact version)

Reply:
ooo good point!!